### ISSUE TITLE
Webview has been removed React Native 

### ISSUE THREAD

**Issue by kiran-mantana** [react-native-community/react-native-webview#818]
**Bug description:**

In My project i have started to migrate from 0.57 to latest RN version, I got issue stating "Webview has been removed React Native  It can now installed and imported from react-native-webview" Attaching the screenshot. 

The issue is i am not using webview directly anywhere instead of it i am using "react-native-wkwebview-reborn", I am not sure why still i am getting this issue.

**To Reproduce:**
After migration, directly when running Android - react-native run-android

**Screenshots/Videos:**
![Screenshot_1567031544](https://user-images.githubusercontent.com/44198435/63898164-6e96a080-ca15-11e9-91cc-9f9a82eb7cb8.png)

**Environment:**
 - OS: Mac OS 
 - OS version: 10.14.6
 - react-native version:0.60.5
 - react-native-webview version: not using

Below are the packages i am using. 

  "dependencies": {
    "@haskkor/react-native-pincode": "^1.21.1",
    "@react-native-community/async-storage": "^1.6.1",
    "@react-native-community/netinfo": "^4.1.5",
    "aws-amplify": "^1.1.32",
    "aws-appsync": "1.8.2-unstable-native.0",
    "aws-appsync-react": "1.2.10-unstable-native.0",
    "babel-polyfill": "^6.26.0",
    "buffer": "^5.2.1",
    "circular-json": "^0.5.7",
    "eslint-config-airbnb": "^17.0.0",
    "eslint-config-prettier": "^2.9.0",
    "eslint-plugin-import": "^2.13.0",
    "eslint-plugin-jsx-a11y": "^6.1.1",
    "eslint-plugin-prettier": "^2.6.2",
    "eslint-plugin-react": "^7.10.0",
    "graphql-tag": "^2.7.3",
    "jest-expo": "~27.0.0",
    "jetifier": "^1.6.3",
    "lottie-ios": "3.0.3",
    "lottie-react-native": "^3.1.0",
    "moment": "^2.22.2",
    "native-base": "^2.13.5",
    "prettier": "^1.13.7",
    "prop-types": "^15.6.2",
    "react": "16.8.6",
    "react-apollo": "^2.1.11",
    "react-native": "^0.60.5",
    "react-native-animatable": "^1.3.0",
    "react-native-audio": "^4.2.2",
    "react-native-autolink": "^1.8.1",
    "react-native-background-upload": "^5.0.0",
    "react-native-blur": "^3.2.2",
    "react-native-brightcove-player": "^1.6.5",
    "react-native-calendars": "^1.211.0",
    "react-native-dialog": "^5.6.0",
    "react-native-document-picker": "^2.3.0",
    "react-native-elements": "^1.1.0",
    "react-native-exit-app": "^1.0.0",
    "react-native-fetch": "^2.0.0",
    "react-native-firebase": "^5.5.6",
    "react-native-flip-card": "^3.5.5",
    "react-native-fs": "^2.14.1",
    "react-native-google-analytics-bridge": "^6.0.1",
    "react-native-highlight-words": "^1.0.1",
    "react-native-htmlview": "^0.13.0",
    "react-native-icon-badge": "^1.1.3",
    "react-native-image-crop-picker": "^0.21.3",
    "react-native-image-gallery": "^2.1.5",
    "react-native-image-picker": "^1.0.2",
    "react-native-interactable": "^0.1.10",
    "react-native-is-device-rooted": "^1.0.3",
    "react-native-loading-spinner-overlay": "^1.0.1",
    "react-native-mime-types": "^2.2.1",
    "react-native-modal": "^11.3.1",
    "react-native-modal-datetime-picker": "^7.5.0",
    "react-native-permissions": "^1.2.0",
    "react-native-popup-dialog": "^0.18.3",
    "react-native-progress": "^3.6.0",
    "react-native-progress-bar-animated": "^1.0.6",
    "react-native-search-filter": "^0.1.4",
    "react-native-searchable-list": "^1.1.3",
    "react-native-snap-carousel": "^3.8.0",
    "react-native-text-input-enhance": "^1.0.11",
    "react-native-touch-id": "^4.4.1",
    "react-native-uuid": "^1.4.9",
    "react-native-vector-icons": "6.6.0",
    "react-native-video": "^5.0.0",
    "react-native-video-controls": "^2.2.3",
    "react-native-wheel-picker-android": "^2.0.5",
    "react-native-wkwebview-reborn": "^2.0.0",
    "react-navigation": "^2.18.2",
    "react-redux": "^7.1.0",
    "redux": "^4.0.2",
    "redux-logger": "^3.0.6",
    "redux-persist": "^5.10.0",
    "redux-thunk": "^2.3.0",
    "rn-fetch-blob": "0.10.14"
  },

**Comment by felipe-cespedes** [comment]
Take a look at your dependencies, in my case it was caused by a dependency called `react-native-gesture-handler`

**Comment by Titozzz** [comment]
What @felipe-cespedes said 🙂

**Comment by Bill-Niz** [comment]
@felipe-cespedes 
> Take a look at your dependencies, in my case it was caused by a dependency called `react-native-gesture-handler`

What did you do than?

**Comment by felipe-cespedes** [comment]
@Bill-Niz I've upgraded the dependency. 

For this particular case, I think that `react-native-wkwebview-reborn` doesn't support the latest version of RN because it's using `WebView`. 
See https://github.com/CRAlpha/react-native-wkwebview/blob/master/WKWebView.android.js

However I saw there is an open PR in `react-native-wkwebview-reborn` 
See https://github.com/CRAlpha/react-native-wkwebview/pull/233/files
So probably RN will be supported by that library in the near future.
